service/azureconfig/v1/resource/resourcegroup/resource.go

```diff
@@ -92,14 +96,10 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
-	tags := map[string]string{
-		clusterIDTag:  key.ClusterID(customObject),
-		customerIDTag: key.ClusterCustomer(customObject),
-	}
 	resourceGroup := Group{
 		Name:     key.ClusterID(customObject),
 		Location: key.Location(customObject),
-		Tags:     tags,
+		Tags:     key.ClusterTags(customObject, r.installationName),
```

Comment:
Here we set the tag for the Resource Group. We have a separate resource group per guest cluster.